frontend/src/composables/useEntriesAPI.ts

```diff
@@ -0,0 +1,29 @@
+import type { Entry } from '@/types/entry'
+
+export function useEntriesAPI() {
+  class EntryAddFailure extends Error {
+    constructor() {
+      super()
+    }
+  }
+
+  const useAddEntryAPI = async (entry: Entry) => {
+    console.log(JSON.stringify(entry))
```

Comment:
ちなみに
```suggestion
    console.log(JSON.stringify(entry, null, 2))
```

をするとみやすいです
https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#%E6%A7%8B%E6%96%87

Reply:
ありがとうございます。